src/pseudopeople/configuration/generator.py

```diff
@@ -179,6 +157,25 @@ def _generate_configuration(is_no_noise: bool) -> ConfigTree:
     return noising_configuration
 
 
+def get_noise_type_dict(noise_type, is_no_noise: bool) -> Dict:
+    noise_type_dict = {}
+    if noise_type.probability is not None:
+        noise_level = 0.0 if is_no_noise else noise_type.probability
+        noise_type_dict[noise_type.probability_key] = noise_level
+    if noise_type.additional_parameters is not None:
+        for key, value in noise_type.additional_parameters.items():
+            # FIXME: This makes a big assumption that the additional parameters are all floats
+            # If we were to add a noise type or additional parameter key that was a list or dict
+            # like we have in some column noise types this would not work.
+            noise_level = (
+                0.0
+                if is_no_noise and noise_type.probability_key == Keys.ROW_PROBABILITY
```

Comment:
A better check here is 
```suggestion
                if is_no_noise and isinstance(noise_type, RowNoiseType)
```